src/components/facility/AcousticDensityTelemetry.tsx

```
@@ -0,0 +1,342 @@
+import AlertOctagon from "lucide-react/dist/esm/icons/alert-octagon";
+import Play from "lucide-react/dist/esm/icons/play";
+
+interface AcousticDensityTelemetryProps {
+  venueId: string;
+}
+
+export function AcousticDensityTelemetry({ venueId }: AcousticDensityTelemetryProps) {
+  const [microphones, setMicrophones] = useState<AcousticMicrophone[]>([]);
+  const [selectedMicId, setSelectedMicId] = useState<string>("");
+  const [telemetry, setTelemetry] = useState<AcousticTelemetry[]>([]);
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)